arm-powerbiembedded/2016-01-29/swagger/powerbiembedded.json

```diff
@@ -0,0 +1,1127 @@
+{
+    "swagger": "2.0",
+    "host": "management.azure.com",
+    "info": {
+        "version": "2016-01-29",
+        "title": "Power BI Embedded Management Client",
+        "description": "Azure Resource Provider for Power BI Embedded"
+    },
+    "schemes": [
+        "https",
+        "http"
+    ],
+    "consumes": [
+        "application/json"
+    ],
+    "produces": [
+        "application/json"
+    ],
+    "paths": {
+        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.PowerBI/workspaceCollections/{workspaceCollectionName}": {
+            "get": {
+                "tags": [
+                    "WorkspaceCollections"
+                ],
+                "operationId": "getWorkspaceCollection",
+                "description": "The **Get Power BI Workspace Collection** operation retrieves an existing Power BI Workspace Collection.",
+                "consumes": [
+                    "application/json"
+                ],
+                "produces": [
+                    "application/json"
+                ],
+                "parameters": [
+                    {
+                        "name": "subscriptionId",
+                        "in": "path",
+                        "description": "Azure subscription id",
+                        "required": true,
+                        "type": "string"
+                    },
+                    {
+                        "name": "resourceGroupName",
+                        "in": "path",
+                        "description": "Azure resource group",
+                        "required": true,
+                        "type": "string"
+                    },
+                    {
+                        "name": "workspaceCollectionName",
+                        "in": "path",
+                        "description": "Power BI Embedded workspace collection name",
+                        "required": true,
+                        "type": "string"
+                    },
+                    {
+                        "name": "api-version",
+                        "in": "query",
+                        "description": "Azure API version",
+                        "required": true,
+                        "type": "string"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "Workspace collection created successfully",
+                        "schema": {
+                            "$ref": "#/definitions/WorkspaceCollection"
+                        }
+                    },
+                    "400": {
```

Comment:
Please make sure to define negative error codes only if you expect the client to not throw an exception on it and deserialize as per the schema of that negative response code.

Take a look at this doc https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/creating-swagger.md#Responses.

Please provide default response with an Error schema to deserialize the Error resposne in that exception